include/qpid/dispatch/router.h

```diff
@@ -33,6 +33,8 @@
 
 #include <stdbool.h>
 
+#define QD_ROUTER_ID_MAX 127  // max length of router id in chars
```

Comment:
It's inconvenient that there is a C version and Python version of the constant. Is the #define actually used anywhere? I can't find any usage in this PR